deployment/tasks/before_symlink.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Copy config file
+  copy: src={{ config_dir }}/{{ inventory_hostname }}/config.prod.json  dest={{ app_dir }}/{{ release_name }}/app/config/config.prod.json
```

Comment:
We could take the opportunity and convert these entries from equal-sing statements to proper YAML, which would increase the readabiliy.